test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -2107,14 +2151,20 @@ static int traffic_mngr_suite_term(void)
 	/* Close the pktios and associated packet pools. */
 	free_rcvd_pkts();
 	for (iface = 0; iface < num_ifaces; iface++) {
-		if (odp_pktio_stop(pktios[iface]) != 0)
-			return -1;
+		/* Skip pktios not initialized */
+		if (pktios[iface] != ODP_PKTIO_INVALID) {
+			if (odp_pktio_stop(pktios[iface]) != 0)
```

Comment:
This may call odp_pktio_stop() for pktios that have not been started, which is against the API spec. Maybe it should be tracked separately whether the pktio has been started.